snoop/data/digests.py

```diff
@@ -117,17 +118,24 @@ def gather(blob, **depends_on):
                 rv['ocrimage'] = True
         rv['ocrtext'] = ocr_results
 
-    # combine texts and detect language
-    if settings.DETECT_LANGUAGE:
+    #TODO: Add entity Extraction setting to snoop settings
```

Comment:
to keep this function small, please move all code to a function inside your module:

```python
    rv.update(entities.extract(all, the, args, you, need, here))
```

and then in your module:

```python3
from django.conf import settings


def extract(bla, bla):
    if settings.DETECT_LANGUAGE:
        ...
```